ros2topic/test/test_info.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from argparse import Namespace
+from contextlib import contextmanager
+from io import StringIO
+import sys
+
+from ros2topic.verb.info import InfoVerb
+
+
+@contextmanager
+def string_stdout() -> StringIO:
+    real_stdout = sys.stdout
+    string_io_stdout = StringIO()
+    sys.stdout = string_io_stdout
+    yield string_io_stdout
+    sys.stdout = real_stdout
```

Comment:
Could this be replaced with [contextlib.redirect_stdout](https://docs.python.org/3.5/library/contextlib.html#contextlib.redirect_stdout)?

Reply:
Fixed.  Log is here: [https://gist.github.com/Nickolaim/6ca58a2454fbd0b4e6530d475106b349](https://gist.github.com/Nickolaim/6ca58a2454fbd0b4e6530d475106b349).